src/components/ReviewSummary/CollectibleReviewSummary.js

```diff
@@ -34,15 +34,17 @@ const Container = styled.View`
   align-items: center;
 `;
 
-const StyledCollectibleImage = styled(CollectibleImage)`
+const CollectibleImageWrapper = styled.View`
   width: 192px;
   height: 192px;
 `;
 
 const CollectibleReviewSummary = ({ collectible, text }: Props) => {
   return (
     <Container>
-      <StyledCollectibleImage source={{ uri: collectible.imageUrl }} />
+      <CollectibleImageWrapper>
+        <CollectibleImage source={{ uri: collectible.imageUrl }} width={192} height={192} />
```

Comment:
If we're defining a  `CollectibleImageWrapper` here - but then defining `width={192} height={192}` - would `CollectibleImage` not expand? Just seems off that we have two elements both with the same height / width definitions...

Reply:
The problem was that SVG images were taking whole screen since it didn't have any wrapper around and we're not setting fixed dimensions on them. With this we have a fixed wrapper and just passing same dimensions to image that might be SVG or not – in case it's not SVG then it takes the passed dimensions to match wrapper. Similar implementation is on Send screen as well.

Reply:
Got it - cheers!